app/models/odk/response_parser.rb

```diff
@@ -178,10 +178,8 @@ def lookup_and_check_form(params)
       raise "xml submissions must be to versioned forms" if form.current_version.nil?
 
       # if form version is outdated, error
-      if form.current_version.code != params[:version] &&
-          form.current_version.number != params[:version]
-        raise FormVersionError, "Form version is outdated"
-      end
+      oldest_accepted_num = form.oldest_accepted_version.number.to_i
```

Comment:
`oldest_accepted_version` is feeling so verbose. Should we change to `minimum_version`? Then cols would be `minimum` and `current` (instead of `is_current`)? Feel free to skip if it would take too long.

Reply:
Significantly over estimate already, so I'll plan to skip this for now, but completely agree that would read better